lib/park.rb

```diff
@@ -0,0 +1,43 @@
+class Park 
+  attr_reader :name, :admission_price, :vehicles, :passengers
+
+  def initialize(name, admission_price)
+    @name = name
+    @admission_price = admission_price
+    @vehicles = []
+  end
+
+  def add_vehicle(vehicle)
+    @vehicles << vehicle
+  end
+
+  def attendees
```

Comment:
```suggestion
  def passengers
```

The interaction pattern asks for this method to be called `passengers`. It seems like a small thing to nitpick, but once you start working on larger projects and teams, it can be pretty important to make sure you're following the "contract" or defined requirements. Not doing so can cause bugs further down the line if folks on other teams are expecting this to be called `passengers` instead.